src/trex/cli/run10x.py

```diff
@@ -195,9 +199,18 @@ def run_trex(
         f"{len(set(clone_ids))} unique)"
     )
 
+    if min_bases_detected > 0:
+        molecules = remove_odd_clone_ids(molecules, min_bases_detected)
```

Comment:
Ah, that would actually be counterproductive. Here is an example. In your test data, there are these reads (columns are cell ID, UMI, cloneID):
```
AAACCCAGTGTCTTAG    AAATTTTACTCA    -------------------------GTGTG
AAACCCAGTGTCTTAG    AAATTTTACTCA    -------------------------GTGTG
AAACCCAGTGTCTTAG    AAATTTTACTCA    ----------------------GTAGTGTG
AAACCCAGTGTCTTAG    AAATTTTACTCA    ----------TAGGCTTTCTGAGTAGTGTG
AAACCCAGTGTCTTAG    AAATTTTACTCA    AGCTCA------------------------
```
Currently, there is no filtering done on these sequences, and then the resulting molecule looks like this:
```
AAACCCAGTGTCTTAG    AAATTTTACTCA    AGCTCA----TAGGCTTTCTGAGTAGTGTG
```
So even though the cloneID was incomplete in many reads, we could still recover most of it in the end.

If the filtering was done on the reads, then this would not work.

Reply:
Yep, after the compute_consensus is done is a good place to filter things out.

My rationale was:
- Very incomplete cloneIDs are compared (in cloneID correction step) but then ignored because they are too incomplete.
- Very incomplete cloneIDs are later looked for in other cells to build clones, but we cannot be sure if they are the same and often such incomplete mulecules might have different number of nucleotides read so they don't match. This adds to noise in Jaccard Indexes.
- Compute_consensus function will yield the most complete version of a cloneID by using reads with the same UMI.

After computing the consensus of the read, if they are still to incomplete, we will not work with them afterwards, so why not filter them in this step?

As we are going to go through the computed molecules in this step, we might as well filter other molecules that are not going to be used: overrepresented ones, too low complexity, and others that we might think of.